app/models/content_configuration.rb

```diff
@@ -23,25 +23,29 @@ class ContentConfiguration < Spree::Preferences::Configuration
   # Producer sign-up page
   # All the following defaults using I18n don't work.
   # https://github.com/openfoodfoundation/openfoodnetwork/issues/3816
-  preference :producer_signup_pricing_table_html, :text,
-             default: I18n.t(:content_configuration_pricing_table)
-  preference :producer_signup_case_studies_html, :text,
-             default: I18n.t(:content_configuration_case_studies)
-  preference :producer_signup_detail_html, :text, default: I18n.t(:content_configuration_detail)
+  # default values for these fields are commented out below
+  preference :producer_signup_pricing_table_html, :text
+  # default: I18n.t(:content_configuration_pricing_table)
+  preference :producer_signup_case_studies_html, :text
+  # default: I18n.t(:content_configuration_case_studies)
+  preference :producer_signup_detail_html, :text
+  # default: I18n.t(:content_configuration_detail)
 
   # Hubs sign-up page
-  preference :hub_signup_pricing_table_html, :text,
-             default: I18n.t(:content_configuration_pricing_table)
```

Comment:
Weird, this key is present:
https://github.com/openfoodfoundation/openfoodnetwork/blob/97985dbe8f118c2a80dae34cd727c6c380a01b6f/config/locales/en.yml#L3182

And it works in the console:

```
[1] pry(main)> I18n.t(:content_configuration_pricing_table)
=> "(TODO: Pricing table)"
```

Reply:
Weird. Maybe because the `preference` is evaluated when the class is loaded, and the locale hasn't been loaded yet?
Hmm that shouldn't be the case though, because I think it's loaded here: https://github.com/openfoodfoundation/openfoodnetwork/blob/master/config/initializers/content_config.rb#L1-L3

Maybe there's something about the context that `to_prepare` runs in.